package.json

```diff
@@ -52,6 +53,7 @@
     "test:system-tests": "lerna run test --scope system-tests --concurrency 1 --stream -- --if-present",
     "test:vscode-insiders-system-tests": "lerna run test:vscode-insiders-system-tests --scope system-tests --concurrency 1 --stream",
     "coverage:system-tests": "lerna exec --scope system-tests --concurrency 1 --stream --bail=false -- npm run coverage:system-tests",
+    "check:deps": "lerna exec --no-bail --stream depcheck",
```

Comment:
Initially is the intention here just to have move information about deps and not to fail or do anything about it? How long does it take? If not very long should be add it to the compile step so we get it for free in the circleCI output?

Reply:
@gbockus-sf Yes this is the first step in getting a checker in. But we need to do more work before this can be integrated into CI. `depcheck` checks for both missing and unused dependencies and it can have false positives. The way I have dealt with this in the past is to use a config file to suppress known false positives.

Here is the output (It only takes a couple of mins):
```
> salesforcedx-vscode@ check:deps /Users/mrajamanickam/src/pdt/salesforcedx-vscode
> lerna exec --no-bail --stream depcheck

salesforcedx-vscode-expanded: No depcheck issue
@salesforce/salesforcedx-sobjects-faux-generator: Running in development mode
@salesforce/salesforcedx-sobjects-faux-generator: Running in development mode
@salesforce/salesforcedx-utils-vscode: Running in development mode
salesforcedx-vscode: No depcheck issue
docs: No depcheck issue
@salesforce/salesforcedx-visualforce-markup-language-server: Unused dependencies
@salesforce/salesforcedx-visualforce-markup-language-server: * vscode-nls
@salesforce/salesforcedx-visualforce-markup-language-server: Unused devDependencies
@salesforce/salesforcedx-visualforce-markup-language-server: * mocha-junit-reporter
@salesforce/salesforcedx-utils-vscode: Unused devDependencies
@salesforce/salesforcedx-utils-vscode: * @salesforce/ts-sinon
@salesforce/salesforcedx-utils-vscode: * @types/mkdirp
@salesforce/salesforcedx-utils-vscode: * @types/shelljs
@salesforce/salesforcedx-utils-vscode: * mocha-junit-reporter
@salesforce/salesforcedx-utils-vscode: Missing dependencies
@salesforce/salesforcedx-utils-vscode: * glob: ./webpack.config.js
@salesforce/salesforcedx-utils-vscode: * vscode: ./test/unit/commands/commandletExecutor.test.ts
@salesforce/salesforcedx-sobjects-faux-generator: Unused devDependencies
@salesforce/salesforcedx-sobjects-faux-generator: * @types/rimraf
@salesforce/salesforcedx-sobjects-faux-generator: * mocha-junit-reporter
@salesforce/salesforcedx-sobjects-faux-generator: * ts-node
@salesforce/salesforcedx-sobjects-faux-generator: Missing dependencies
@salesforce/salesforcedx-sobjects-faux-generator: * glob: ./webpack.config.js
@salesforce/salesforcedx-sobjects-faux-generator: * lolex: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * @sinonjs/formatio: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * path-to-regexp: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * @sinonjs/text-encoding: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * just-extend: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * type-detect: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * @sinonjs/samsam: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * @sinonjs/commons: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * lodash: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-sobjects-faux-generator: * isarray: ./out/test/integration/sObjectDescribe.test.js
@salesforce/salesforcedx-test-utils-vscode: Unused devDependencies
@salesforce/salesforcedx-test-utils-vscode: * @types/chai
@salesforce/salesforcedx-test-utils-vscode: * @types/sinon
@salesforce/salesforcedx-test-utils-vscode: * chai
@salesforce/salesforcedx-test-utils-vscode: * mocha-junit-reporter
@salesforce/salesforcedx-test-utils-vscode: * mocha-multi-reporters
@salesforce/salesforcedx-test-utils-vscode: * mock-spawn
@salesforce/salesforcedx-test-utils-vscode: * nyc
@salesforce/salesforcedx-test-utils-vscode: * request-light
@salesforce/salesforcedx-test-utils-vscode: * sinon
@salesforce/salesforcedx-visualforce-language-server: Unused devDependencies
@salesforce/salesforcedx-visualforce-language-server: * mocha-junit-reporter
@salesforce/salesforcedx-apex-replay-debugger: Unused devDependencies
@salesforce/salesforcedx-apex-replay-debugger: * mocha-junit-reporter
salesforcedx-vscode-soql: Running in development mode
salesforcedx-vscode-visualforce: Unused dependencies
salesforcedx-vscode-visualforce: * @salesforce/salesforcedx-visualforce-language-server
salesforcedx-vscode-visualforce: * @salesforce/salesforcedx-visualforce-markup-language-server
salesforcedx-vscode-visualforce: * vscode-languageserver-types
salesforcedx-vscode-visualforce: Unused devDependencies
salesforcedx-vscode-visualforce: * @types/mocha
salesforcedx-vscode-visualforce: * mocha
salesforcedx-vscode-visualforce: Missing dependencies
salesforcedx-vscode-visualforce: * vscode: ./test/vscode-integration/fileAssociations.test.ts
salesforcedx-vscode-lightning: Unused dependencies
salesforcedx-vscode-lightning: * @salesforce/aura-language-server
salesforcedx-vscode-lightning: * @salesforce/core
salesforcedx-vscode-lightning: * applicationinsights
salesforcedx-vscode-lightning: * vscode-extension-telemetry
salesforcedx-vscode-lightning: Unused devDependencies
salesforcedx-vscode-lightning: * @types/mkdirp
salesforcedx-vscode-lightning: * @types/open
salesforcedx-vscode-lightning: * @types/sinon
salesforcedx-vscode-lightning: * @types/which
salesforcedx-vscode-lightning: * mocha-junit-reporter
salesforcedx-vscode-lightning: * sinon
salesforcedx-vscode-lightning: Missing dependencies
salesforcedx-vscode-lightning: * tslint: ./test/tslint.json
salesforcedx-vscode-lightning: * vscode: ./test/vscode-integration/extension.test.ts
salesforcedx-vscode-apex-replay-debugger: Unused devDependencies
salesforcedx-vscode-apex-replay-debugger: * @salesforce/ts-sinon
salesforcedx-vscode-apex-replay-debugger: * @types/mkdirp
salesforcedx-vscode-apex-replay-debugger: * @types/mocha
salesforcedx-vscode-apex-replay-debugger: * mocha
salesforcedx-vscode-apex-replay-debugger: Missing dependencies
salesforcedx-vscode-apex-replay-debugger: * vscode: ./test/vscode-integration/index.test.ts
system-tests: Unused devDependencies
system-tests: * @salesforce/salesforcedx-utils-vscode
system-tests: * @types/webdriverio
system-tests: * decache
system-tests: * electron
system-tests: * istanbul
system-tests: * mocha-junit-reporter
system-tests: * mocha-multi-reporters
system-tests: * remap-istanbul
system-tests: Missing dependencies
system-tests: * vscode: ./test/index.ts
system-tests: * lwc: ./assets/sfdx-simple/force-app/main/default/lwc/demoLwcComponent/demoLwcComponent.js
system-tests: * c: ./assets/sfdx-simple/force-app/main/default/lwc/demoLwcComponent/__tests__/demoLwcComponent.test.js
salesforcedx-vscode-apex: Unused dependencies
salesforcedx-vscode-apex: * @salesforce/apex-tmlanguage
salesforcedx-vscode-apex: * vscode-extension-telemetry
salesforcedx-vscode-apex: Unused devDependencies
salesforcedx-vscode-apex: * @salesforce/ts-sinon
salesforcedx-vscode-apex: * @types/mkdirp
salesforcedx-vscode-apex: * @types/mocha
salesforcedx-vscode-apex: * mocha
salesforcedx-vscode-apex: * vscode-extension-telemetry
salesforcedx-vscode-apex: Missing dependencies
salesforcedx-vscode-apex: * vscode: ./test/vscode-integration/javaConfiguration.test.ts
@salesforce/salesforcedx-apex-debugger: Unused devDependencies
@salesforce/salesforcedx-apex-debugger: * mocha-junit-reporter
salesforcedx-vscode-lwc: Unused dependencies
salesforcedx-vscode-lwc: * @salesforce/core
salesforcedx-vscode-lwc: * @salesforce/eslint-config-lwc
salesforcedx-vscode-lwc: * @salesforce/lwc-language-server
salesforcedx-vscode-lwc: * ajv
salesforcedx-vscode-lwc: * applicationinsights
salesforcedx-vscode-lwc: * eslint
salesforcedx-vscode-lwc: * vscode-extension-telemetry
salesforcedx-vscode-lwc: Unused devDependencies
salesforcedx-vscode-lwc: * @types/mkdirp
salesforcedx-vscode-lwc: * mocha-junit-reporter
salesforcedx-vscode-lwc: Missing dependencies
salesforcedx-vscode-lwc: * tslint: ./test/tslint.json
salesforcedx-vscode-lwc: * vscode: ./test/vscode-integration/hover.test.ts
salesforcedx-vscode-soql: Unused dependencies
salesforcedx-vscode-soql: * @salesforce/apex-tmlanguage
salesforcedx-vscode-soql: * @salesforce/soql-builder-ui
salesforcedx-vscode-soql: * @salesforce/soql-data-view
salesforcedx-vscode-soql: Unused devDependencies
salesforcedx-vscode-soql: * @types/mkdirp
salesforcedx-vscode-soql: * @types/mocha
salesforcedx-vscode-soql: * applicationinsights
salesforcedx-vscode-soql: * webpack-cli
salesforcedx-vscode-soql: Missing dependencies
salesforcedx-vscode-soql: * eslint-config-prettier: ./.eslintrc.json
salesforcedx-vscode-soql: * vscode: ./test/vscode-integration/commomUtils.test.ts
salesforcedx-vscode-core: Unused dependencies
salesforcedx-vscode-core: * @salesforce/schemas
salesforcedx-vscode-core: * applicationinsights
salesforcedx-vscode-core: * yeoman-generator
salesforcedx-vscode-core: Unused devDependencies
salesforcedx-vscode-core: * @salesforce/ts-sinon
salesforcedx-vscode-core: * @types/mkdirp
salesforcedx-vscode-core: * mocha-junit-reporter
salesforcedx-vscode-core: * mocha-multi-reporters
salesforcedx-vscode-core: * nyc
salesforcedx-vscode-core: Missing dependencies
salesforcedx-vscode-core: * vscode: ./test/vscode-integration/util/authInfo.test.ts
salesforcedx-vscode-apex-debugger: Unused devDependencies
salesforcedx-vscode-apex-debugger: * @types/mocha
salesforcedx-vscode-apex-debugger: * mocha
salesforcedx-vscode-apex-debugger: Missing dependencies
salesforcedx-vscode-apex-debugger: * vscode: ./test/vscode-integration/index.test.ts
```

Reply:
word.  a couple of minutes is a lot for every compile so cool with just having it available for now 👍